src/main/java/com/iota/iri/SignedFiles.java

```diff
@@ -22,21 +20,29 @@
     private static final Logger log = LoggerFactory.getLogger(SignedFiles.class);
 
     public static boolean isFileSignatureValid(String filename, String signatureFilename, String publicKey, int depth, int index) {
-        int[] trits = new int[Curl.HASH_LENGTH*3];
+        int[] trits = new int[Curl.HASH_LENGTH * 3];
         Sponge curl = SpongeFactory.create(SpongeFactory.Mode.KERL);
-            //digest file
-            File file = new File(filename);
-            try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
-                String line;
-                while ((line = reader.readLine()) != null) {
-                    Converter.trits(Converter.asciiToTrytes(line), trits, 0);
-                    curl.absorb(trits, 0, trits.length);
-                    Arrays.fill(trits, 0);
-                }
-            } catch (IOException e) {
-                log.error("Can't read file " + filename, e);
-                return false;
+        BufferedReader reader = null;
+        //digest file
+        try {
+            InputStream inputStream = SignedFiles.class.getResourceAsStream(filename);
+            if (inputStream == null) {
+                inputStream = new FileInputStream(filename);
```

Comment:
add a comment to explicitly state that if a path is not in resources, then reading the file from OS.

Reply:
Added similar comment in `Snapshot.java` as well